frontend.py

```diff
@@ -49,11 +55,15 @@ def my_form():
 @cross_origin(origin='*')
 def my_form_post():
     username = request.form['text']
+    real_name = 'no' 
+    if (check_for_real_name(usermame)):    # Function returns bool
+        real_name = 'yes'
     return render_template('status.html',
                            username=username,
                            sites=sites,
                            signup=signup,
                            logos=logos,
+                           real_name=real_name,
```

Comment:
you add a parameter to the template invocation, but ... the template isnt modified in this PR.  Perhaps you forgot to add your template changes to this commit ?